ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -149,6 +152,13 @@ public void executeRequest(String method, String routeName, String endpoint, Str
         lastResponse = getInvocation(url).method(method, Entity.entity(body, MediaType.APPLICATION_JSON_TYPE));
     }
 
+    @Then("^verify that executing ([A-Z]+) on API Provider route ([\\w-]+) endpoint \"([^\"]*)\" returns status (\\d+) and body$")
```

Comment:
I don't understand, why are we adding a deprecated step? Also, why is there no assert in a step that says `verify...`?